cheatsheets_draft/Cookie_Theft_Mitigation.md

```diff
@@ -0,0 +1,132 @@
+# Cookie Theft Mitigation Cheat Sheet
+
+## Introduction
+
+With the spread of 2FA and Passkey, the login process has become more robust, and even if an attacker steals only the password, it has become difficult to do a spoofing attack.
+
+However, if attacker can steal a valid session cookie instead, it is possible to hijack the user session for the duration of the session lifetime period. In other words, stealing a session cookie has the same impact as stealing authentication credentials until it's expires. No matter how robust your authentication process is, it will not be a sufficient countermeasure for Cookie Theft.
```

Comment:
```suggestion
However, if attacker can steal a valid session cookie instead, it is possible to hijack the user session for the duration of the session lifetime period. In other words, stealing a session cookie has the same impact as stealing authentication credentials until it expires. No matter how robust your authentication process is, it will not be a sufficient countermeasure for Cookie Theft.
```